File: examples/swift/constructs/scope_call.swift
// Ported from examples/lumen/constructs/scope_call.lm by scripts/port_examples.py; edit the Lumen original, not this file.
var k = 1
func show() {
    print(k)
}

func caller() {
    let k = 5
    show()
    print(k)
}

caller()
print(k)
show()

File: examples/python/constructs/scope_call.py
# Ported from examples/lumen/constructs/scope_call.lm by scripts/port_examples.py; edit the Lumen original, not this file.
k = 1
def show():
    print(k)

def caller():
    k = 5
    show()
    print(k)

caller()
print(k)
show()
